docs/resource-sharing/os-backfill-containers.md

```diff
@@ -27,34 +27,145 @@ In order to configure the container, you will need:
+   `-v /path/to/token:/etc/condor/tokens-orig.d/flock.opensciencegrid.org`.
+   (Replace `/path/to/token` with the full path to the token you obtained from the OSP administrators.)
+2. Set `GLIDEIN_Site` and `GLIDEIN_ResourceName` to match the site name and resource name that you registered in Topology,
    respectively.
-3. Set the `OSG_SQUID_LOCATION` environment variable to the HTTP address of your preferred squid instance.
-4. _Optional:_  Some sites prefer that job I/O is done in a specific temporary directory instead of inside the container.
-   To do this, map the appropriate directory on the host to `/pilot` inside containers.
-   If you are using Docker to launch the container, this is done with the command line flag `-v /somelocaldir:/pilot`.
-5. _Optional:_ add an expression with the `GLIDEIN_Start_Extra` environment variable to append to the HTCondor `START`
+3. Set the `OSG_SQUID_LOCATION` environment variable to the HTTP address of your preferred Squid instance.
+4. _Recommended:_ Enable [CVMFS](#cvmfs) via one of the mechanisms described below.
+5. _Optional:_  If you want job I/O to be done in a separate directory outside of the container,
+   volume mount the desired directory on the host to `/pilot` inside the container.
+   If you are using Docker to launch the container, this is done with the command line flag
+   `-v /somelocaldir:/pilot`.
+6. _Optional:_ add an expression with the `GLIDEIN_Start_Extra` environment variable to append to the HTCondor `START`
    expression; this limits the pilot to only run certain jobs.
 
 Here is an example invocation using `docker run` by hand:
 
 ```
-docker run -it --rm --user osg \
-       --cap-add=DAC_OVERRIDE --cap-add=SETUID --cap-add=SETGID \
-       --cap-add=SYS_ADMIN --cap-add=SYS_CHROOT --cap-add=SYS_PTRACE \
-       --cap-add=CAP_DAC_READ_SEARCH \
-       -v /cvmfs:/cvmfs:shared \
-       -e TOKEN="..." \
-       -e GLIDEIN_Site="..." \
-       -e GLIDEIN_ResourceName="..." \
-       -e GLIDEIN_Start_Extra="True" \
-       -e OSG_SQUID_LOCATION="..." \
+docker run -it --rm --user osg  \
+       --privileged             \
+       -v /path/to/token:/etc/condor/tokens-orig.d/flock.opensciencegrid.org \
+       -e GLIDEIN_Site="..."                    \
+       -e GLIDEIN_ResourceName="..."            \
+       -e GLIDEIN_Start_Extra="True"            \
+       -e OSG_SQUID_LOCATION="..."              \
+       -e CVMFSEXEC_REPOS="                     \
+            oasis.opensciencegrid.org           \
+            singularity.opensciencegrid.org"    \
        opensciencegrid/osgvo-docker-pilot:release
 ```
 
-Note the additional capabilities requested in the above `docker run` allow the container to invoke `singularity` for
-user jobs; this allows the user to utilize a container for their job that is different from the pilot.
+Replace `/path/to/token` with the location you saved the token obtained from the OSG VO administrators.
```

Comment:
You dropped the parens here but did not update the phrasing like in the other location (OSP, etc.)